packages/appstore/src/components/app.tsx

```diff
@@ -4,6 +4,7 @@ import CFDStoreProvider from '@deriv/cfd/src/cfd-providers';
 import { StoreProvider } from '@deriv/stores';
 import AppContent from './app-content';
 import './app.scss';
+import { APIProvider } from '@deriv/api';
```

Comment:
```suggestion
```
Do not use any packages from the `@deriv/api` hooks! This wont work well with MobX providers.

Reply:
This also can be removed as we are not using any hooks from @deriv/api now